cli/src/serve.js

```diff
@@ -229,6 +229,11 @@ const initialize_servers = (ctor, opts) => {
           console.info(`Static files being served from ${opts.serve_static}`);
         }
         srv.on('request', fileServer(opts.serve_static));
+      } else {
```

Comment:
Ah, that code path no longer works since we're using `engine.io` - it installs its own request handler to pass things down (and it doesn't 404 if there's nothing there).  Since we're thinking of moving away from `engine.io`, I don't see the problem with leaving the handling in both the `cli` and `server`.